ffi/rtc.lua

```diff
@@ -0,0 +1,196 @@
+--[[--
+Module for interfacing with the RTC (real time clock).
+
+This module provides the ability to schedule wakeups through RTC.
+
+@module ffi.rtc
+]]
+
+local ffi = require("ffi")
+local bor = bit.bor
+local C = ffi.C
+
+local dummy = require("ffi/posix_h")
+local dummy = require("ffi/rtc_h")
```

Comment:
(remind me) what does these do, are they needed?

Reply:
We need them to add the requisite functions & constants to the ffi C. namespace ;).

Reply:
Might be worth a comment (a quick check at base/ffi/* shows no comment around these, so your sentence deserves to be at least in one file :)

Reply:
Not true :-P

https://github.com/koreader/koreader-base/blob/4327fd094cef238eed00d9b897650b3c0f8fc144/ffi/kobolight.lua#L9-L10

Reply:
@NiLuJe sentence above is just perfect and more precise.

Reply:
@poire-z Something like this?

Reply:
yep, fine, thanks